### PR TITLE
Fix N-06

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Fixes https://defender.openzeppelin.com/#/audit/0ed5cb02-a980-4c27-8890-d74c15e0438c/issues/N-06

### CODE REVIEW

**Review by MathisGD — APPROVED** [review]

**Review by colin-morpho — APPROVED** [review]

File: src/MorphoTokenEthereum.sol
Comment (QGarchery) [inline src/MorphoTokenEthereum.sol:1]: We can remove the following line as well:
```solidity
require(owner != address(0), ZeroAddress());
```

**Review by QGarchery — CHANGES_REQUESTED** [review]
We

File: src/MorphoTokenEthereum.sol
Comment (colin-morpho) [reply src/MorphoTokenEthereum.sol:1]: Implemented in https://github.com/morpho-org/morpho-token-upgradeable/pull/70/commits/ee003565bfbd692c5ced781661bd223cf6689917.

**Review by colin-morpho — APPROVED** [review]

File: src/MorphoTokenEthereum.sol [hunk 1]
```diff
@@ -27,12 +27,10 @@ contract MorphoTokenEthereum is DelegationToken {
     /// @param owner The new owner.
     /// @param wrapper The wrapper contract address to migrate legacy MORPHO tokens to the new one.
     function initialize(address owner, address wrapper) external initializer {
-        require(owner != address(0), ZeroAddress());
```
Comment (MerlinEgalite) [inline src/MorphoTokenEthereum.sol:30]: The error can be removed I think

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]

File: src/MorphoTokenEthereum.sol (hunk 1 above)
Comment (MerlinEgalite) [reply src/MorphoTokenEthereum.sol:30]: wait just seeing [this](https://github.com/morpho-org/morpho-token-upgradeable/pull/70#discussion_r1819398451) but i can't find the changes

**Comment by MerlinEgalite** [comment]
branch name is fix/n-05 while it's fixing N-067 which is super confusing

**Comment by Jean-Grimal** [comment]
> branch name is `fix/n-05` while it's fixing N-067 which is super confusing

Yes I did a mistake in the naming

File: src/MorphoTokenEthereum.sol
Comment (QGarchery) [inline src/MorphoTokenEthereum.sol:1]: Let's also remove the `ZeroAddress` error in this contract

**Review by QGarchery — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Review by MathisGD — APPROVED** [review]

**Pull request merged**